test/specs/collections/Breadcrumb/Breadcrumb-test.js

```diff
@@ -12,14 +12,29 @@ describe('Breadcrumb', () => {
       .should.have.tagName('div')
   })
 
+  const sections = [
+    { text: 'Home', link: true },
+    { text: 'T-Shirt', href: 'google.com' },
+  ]
+
   it('renders children with `sections` prop', () => {
-    const sections = [
-      { text: 'Home', link: true },
-      { text: 'T-Shirt', href: 'google.com' },
-    ]
     const wrapper = shallow(<Breadcrumb sections={sections} />)
 
     wrapper.find('BreadcrumbDivider').should.have.length(1)
     wrapper.find('BreadcrumbSection').should.have.length(2)
   })
+
+  it('renders defined divider with `divider` prop', () => {
+    const wrapper = mount(<Breadcrumb sections={sections} divider='>' />)
+    const divider = wrapper.find('BreadcrumbDivider').first()
+
+    divider.text().should.to.equal('>')
+  })
+
+  it('renders divider as `Icon` with `icon` prop', () => {
```

Comment:
Since `icon` and `image` props should behave consistently across the entire project, there is a common tests for them.  You can remove this test and replace it with:

```js
common.implementsIconProp(Breadcrumb)
```